integration/localnet/README.md

```
@@ -120,31 +157,38 @@ Here's an example of a Prometheus query that filters by the `consensus` role:
 avg(rate(consensus_compliance_finalized_blocks_total{role="consensus"}[$__interval]))
 ```
 
-## Traces
+### Traces
 Traces are available through the Tempo backend.
-You can to traces either by searching for logs that have a `traceID` label, clicking on them and pressing "Open in Tempo" button: 
+You can get to traces either by searching for logs that have a `traceID` label, clicking on them and pressing "Open in Tempo" button: 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)